plone/app/locales/locales/pt_BR/LC_MESSAGES/plone.po

```diff
@@ -14603,12 +14600,12 @@ msgstr "Clique para visualizar o tema aplicado. Formulários serão desabilitado
 #. Default: "Translations:"
 #: plone/app/multilingual/browser/templates/mmap.pt:119
 msgid "translation"
-msgstr ""
+msgstr "Traduções."
```

Comment:
"Traduções:" ao invés de "Traduções."

Reply:
what? that's the same.

Reply:
Nope. You're using a dot `.` instead of a `:` after the word.